tests/__sf_section_dir.test.fish

```diff
@@ -307,3 +316,90 @@ test "Changing SPACEFISH_DIR_COLOR changes the dir color"
 		set_color normal
 	) = (__sf_section_dir)
 end
+
+
+#
+# SPACEFISH_DIR_LOCK_SYMBOL
+#
+
+test "Shows DIR_LOCK_SYMBOL if in a dir with no write permissions and SPACEFISH_DIR_LOCK_SHOW is true"
+	(
+		set SPACEFISH_DIR_LOCK_SHOW $IS_NOT_CYGWIN
+		cd /tmp/tmp-spacefish
+		mkdir testDir
+		chmod 500 testDir
+		cd testDir/
+
+		set_color --bold fff
+		echo -n "in "
+		set_color normal
+		set_color --bold cyan
+		echo -n "tmp/tmp-spacefish/testDir"
+		set_color normal
+		set_color --bold fff
+		if [ $SPACEFISH_DIR_LOCK_SHOW = "true" ]
+			echo -n (set_color red)" "(set_color --bold fff)
+		end
+		echo -n " "
+		set_color normal
+	) = (__sf_section_dir)
+end
+
+test "Doesn't show DIR_LOCK_SYMBOL if SPACEFISH_DIR_LOCK_SHOW is false"
```

Comment:
Oops! Looks like this test wasn't written yet 😮

Reply:
oh! 🙈